public/src/components/calls/calls_page_parent.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+import { Row, Col } from 'react-bootstrap';
+import CallsSideBar from './calls_sidebar';
+import CallsPageMain from './calls_page_main';
+
+export default class CallPage extends Component {
+  componentDidMount() {
+    console.log('#############meow', this.props);
```

Comment:
do you want to keep this console.log in here?

Reply:
and uncomment the below props?

Reply:
Yeah, just for "dummy purposes," the whole component will essentially be rewritten, this was kind of a "proof of concept" that the correct props were accessible.